web/src/hotkey.js

```diff
@@ -716,6 +718,13 @@ export function process_hotkey(e, hotkey) {
 
     // `list_util` will process the event in send later modal.
     if (modals.any_active() && modals.active_modal() !== "#send_later_modal") {
+        if (
+            event_name === "toggle_read_receipts" &&
+            modals.active_modal() === "#read_receipts_modal"
+        ) {
+            read_receipts.hide_user_list();
+            return true;
+        }
         return false;
     }
```

Comment:
Let's at least make the outer block `modals.any_active()` and then have a couple `if` blocks inside; `modals.any_active` queries the DOM, so we don't want to do it multiple times if we don't need to.

Reply:
I ended up storing the values returned by `modals.any_active()` and `modals.active_modal()` as constants since both of these query the DOM. (`modals.active_modal()` calls `modals.any_active()` inside it's definition, which in turn queries the DOM). It made sense to store it as constants since the active modal and the modal state doesn't change in between the processing of the hotkeys.